stat/readme.txt

```diff
@@ -0,0 +1,4 @@
+Zajímavým úkazem je, že pokud se zvolí šikovnné "seed number", tak lineární model dává přesnější odhady než
+neuronová síť. Pozoroval jsem to asi dvakrát, kdy mi to vyšlo úplně náhodou. Otázkou je, zda tedy neudělat sérii
+běhů (třeba 100), přičemž při každém by se náhodně vybrala testovací a trénovací data z celého datasetu a pak se 
+statisticky porovnávaly odchylky. To by podle mého byla asi metodicky nejzajímavější varianta.
```

Comment:
Určitě zajímavý nápad.
Nemám zkušenosti s lineárním modelem, ale obecně ta neuronová síť se mi jeví jako obrovský kanon. S nasim malym datasetem to upravdu může být spíš náhoda...